pkg/verification/verification_service.go

```diff
@@ -130,8 +133,11 @@ func (s *ServiceImpl) VerifyCode(ctx *gin.Context, signup *v1alpha1.UserSignup,
 	}
 
 	if now.After(exp) {
-		// If it is now past the expiry timestamp for the verification code, return a 400 Bad Request error
-		return errors2.NewBadRequest("verification code expired")
+		// If it is now past the expiry timestamp for the verification code, return a 403 Forbidden error
+		return errors2.NewForbidden(schema.GroupResource{
```

Comment:
I'm curious, why do you return a kube error? Wouldn't some http error be more suitable?

Reply:
Good point.  For some reason I was thinking these were gin errors, however it turns out there are no third party http errors besides the kube ones.  For this reason I've added a few new error constructors in the error package.